recipes/mod_wsgi/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "mod_wsgi" %}
+{% set version = "4.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 64a6a177946b78bb1eb8ecb31fb3aea5f6d306c4eccd247433563ed6afceafe5
+
+build:
+  skip: true  # [py27]
```

We don't build for Python 2.7 any more so this can be:

```suggestion
  skip: true  # [win]
```

---

done and also updated to 4.7.1